app/javascript/packs/banner_editor.jsx

```diff
@@ -324,8 +338,8 @@ export default class BannerEditor extends React.Component {
     this.setState({ linkOption: value });
   }
 
-  updateCurrentLink(event) {
-    this.setState({ currentLink: event.target.value });
+  updateCurrentUsername(event) {
+    this.setState({ currentUsername: event.target.value.replace("/\W/g", '') });
```

Comment:
Do we want to filter all non-word characters, I know some twitter usernames include underscores

Reply:
Ah didn't know. I'll fix.